source/manage_the_buzz_for_developers/applications_and_user_system_manage_the_buzz/custom_solution_for_self_managed_user_system/quick_start_guide_service_apps.rst

```diff
@@ -3,9 +3,34 @@
 Quick Start Guide - Service Apps
 ================================
 
-.. figure:: /_static/coming-soon.png
-   :align: center
+Register Application
+++++++++++++++++++++
 
-   **Coming Soon**
+1. Signup for a new COCO account or Sign In if you have one: https://manage.getcoco.buzz/
+2. Click on Applications > (+) Application
+3. Select application type as **Service** and fill in the details
+
+.. image:: ../../../_static/registerApp.png
+
+4. For grouping of applications, select existing app group or create a new one.
```

Comment:
Mention this in step 3 itself as the second sentence.

Reply:
Updated